frontend/app/services/subscription-service.server.ts

```diff
@@ -102,5 +102,42 @@ function createSubscriptionService() {
     }
   }
 
-  return { getSubscription, updateSubscription };
+  async function validateConfirmationCode(userEmail: string, enteredConfirmationCode: string, sin: string) {
+    const auditService = getAuditService();
+    const instrumentationService = getInstrumentationService();
+
+    auditService.audit('alert-subscription.validate', { sin });
```

Comment:
I missed this in previous PR in this file for `getSubscription(..)` above, but we shouldn't be logging the SIN but rather the `sub` claim of RAOIDC's `idToken`.

Reply:
sin has been changed to the sub